cmd/init.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/thi-startup/spitfire/internal/get"
+	"github.com/thi-startup/spitfire/utils"
+
+	log "github.com/charmbracelet/log"
+)
+
+func MakeInitCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:          "init",
+		Short:        "download assets if they don't exist",
```

Comment:
- i don't think an init command should be downloading stuff from the internet. the name does not imply that, command should imply the action they perform, so pull/download will be better for things that pull from the internet. you know where my mind is at

Reply:
I agree. What name would you suggest though. The idea is that it's a command that makes sure that all dependencies are met